docs/book/features/file_volume.md

```diff
@@ -0,0 +1,18 @@
+# File volumes
+
+The vSphere Vanilla CSI driver version 2.0 and above supports file volumes backed by vSAN File shares using NFSv4 protocol to be statically/dynamically created and mounted by stateful containerized applications. This feature allows you to reference the same shared data among multiple pods spread across different clusters using the `ReadWriteMany` and `ReadOnlyMany` access modes making it an absolute necessity for applications that need shareability.
+
+Before you setup file services in your environment keep in mind that if you have file shares being shared across more than one clusters in your vCenter, deleting a PVC with reclaim policy set to `Delete` in any one cluster may delete the underlying file share causing the volume to be unavailable for the rest of the clusters.
+
+Also note that the file volumes feature doesn't work in conjunction with the topology aware zones, extend volume and encryption features.
+
+Proceed to the requirements section below to enable this feature in your environment. After you setup file services, to get started with vSphere CSI file services integration with your applications, check this [video](https://youtu.be/GUtG-4urGFA). We have also provided few [Try-out YAMLs](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/example/vanilla-k8s-file-driver) for Storage Class, PVC, Pods for your convenience.
```

Comment:
it is ok to put a try out yaml link, but this feature needs to be explained somewhere.

Can you highlight somewhere what `accessModes` user need to set in the PVC to provision file volume?

I think yamls needs to be explained somewhere. just like we are explaining in detail for block volume, resize and volume topology.

Reply:
Done